tests/test_formats.py

```diff
@@ -1060,6 +1062,511 @@ def test_format_number_with_sep_dec_marks():
+        decimals=decimals,
+        drop_trailing_dec_mark=drop_trailing_dec_mark,
+        use_seps=use_seps,
+        placement=placement,
+        incl_space=incl_space,
+    )
+    x = _get_column_of_values(gt, column_name="x", context="html")
+    assert x == x_out
+
+
+@pytest.mark.parametrize(
+    "force_sign,x_out",
+    [
+        (
+            False,
+            [
+                "−$234.65",
+                "−$0.00",
+                "−$0.10",
+                "$0.00",
+                "$2,352.23",
+                "$12,354.30",
+                "$9,939,293,923.23",
+            ],
+        ),
+        (
+            True,
+            [
+                "−$234.65",
+                "−$0.00",
+                "−$0.10",
+                "$0.00",
+                "+$2,352.23",
+                "+$12,354.30",
+                "+$9,939,293,923.23",
+            ],
+        ),
+    ],
+)
+def test_fmt_currency_force_sign(force_sign: bool, x_out: str):
+    df = pd.DataFrame({"x": [-234.654, -0.000634, -0.1, 0, 2352.23, 12354.3, 9939293923.23]})
+
+    gt = GT(df).fmt_currency(columns="x", force_sign=force_sign)
+    x = _get_column_of_values(gt, column_name="x", context="html")
+    assert x == x_out
+
+
+# ------------------------------------------------------------------------------
+# Test `fmt_time()`
+# ------------------------------------------------------------------------------
+
+
+@pytest.mark.parametrize(
+    "time_style,x_out",
+    [
+        (
+            "iso",
+            ["00:00:00", "08:30:00", "10:59:59", "12:00:00", "13:23:59", "22:10:30", "23:15:00"],
+        ),
+        ("iso-short", ["00:00", "08:30", "10:59", "12:00", "13:23", "22:10", "23:15"]),
+        (
+            "h_m_s_p",
+            [
+                "12:00:00 AM",
+                "8:30:00 AM",
+                "10:59:59 AM",
+                "12:00:00 PM",
+                "1:23:59 PM",
+                "10:10:30 PM",
+                "11:15:00 PM",
+            ],
+        ),
+        (
+            "h_m_p",
+            ["12:00 AM", "8:30 AM", "10:59 AM", "12:00 PM", "1:23 PM", "10:10 PM", "11:15 PM"],
+        ),
+        ("h_p", ["12 AM", "8 AM", "10 AM", "12 PM", "1 PM", "10 PM", "11 PM"]),
+    ],
+)
+def test_fmt_time(time_style: TimeStyle, x_out: str):
+    df = pd.DataFrame(
+        {
+            "x": [
+                "00:00:00",
+                "08:30:00",
+                "10:59:59",
+                "12:00:00",
+                "13:23:59",
+                "22:10:30",
+                "23:15",
+            ]
+        }
+    )
+
+    gt = GT(df).fmt_time(columns="x", time_style=time_style)
+    x = _get_column_of_values(gt, column_name="x", context="html")
+    assert x == x_out
+
+
+# ------------------------------------------------------------------------------
+# Test `fmt_datetime()`
+# ------------------------------------------------------------------------------
+
+
+@pytest.mark.parametrize(
```

Comment:
Same feedback as above, on splitting into single input cases. It appears the underlying logic is largely babel's format_datetime, so you don't need to re-test that (but it makes sense there's a lot of stitching of arguments to fmt_datetime that need to get tested).